packages/amplify-graphql-model-transformer/src/__tests__/__snapshots__/model-transformer-override.test.ts.snap

```diff
@@ -766,6 +766,7 @@ $util.toJson({})",
                 "dynamodb:PutItem",
                 "dynamodb:UpdateItem",
                 "dynamodb:DeleteItem",
+                "dynamodb:DescribeTable",
```

Comment:
Do we know why this policy is updated?

Reply:
I think this is related to the cdk dependency change. This seems like an auto-added action by the cdk API as this is not included in the [code](https://github.com/aws-amplify/amplify-category-api/blob/main/packages/amplify-graphql-model-transformer/src/graphql-model-transformer.ts#L1335-L1344). I check the doc [here](https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_policies_examples_dynamodb_specific-table.html) and this action is included. I don't have any strong opinion to not include this in the policy